WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -77,10 +80,21 @@ public static void savePeople(List<Person> peopleList) {
         }
     }
 
+    public static void deletePeopleForBlog(int localTableBlogId) {
+        String[] args = new String[]{Integer.toString(localTableBlogId)};
+        getWritableDb().delete(PEOPLE_TABLE, "local_blog_id=?", args);
+    }
+
+    public static void deletePerson(long personID, int localTableBlogId) {
+        String[] args = new String[]{Long.toString(personID), Integer.toString(localTableBlogId)};
+        getWritableDb().delete(PEOPLE_TABLE, "person_id=? AND local_blog_id=?", args);
+    }
+
     public static List<Person> getPeople(int localTableBlogId) {
         List<Person> people = new ArrayList<>();
         String[] args = { Integer.toString(localTableBlogId) };
-        Cursor c = getReadableDb().rawQuery("SELECT * FROM " + PEOPLE_TABLE + " WHERE local_blog_id=?", args);
+        Cursor c = getReadableDb().rawQuery("SELECT * FROM " + PEOPLE_TABLE +
+                " WHERE local_blog_id=? ORDER BY display_name, user_name", args);
```

Comment:
Not in this changeset but, there are a couple of lines below this one that could use the same treatment and be wrapped at 120chars. Cheers!